content/blog/2021-06-25-june-21-community-gems.md

```diff
@@ -0,0 +1,121 @@
+---
+title: June '21 Community Gems
+date: 2021-06-25
+description: |
+  A roundup of technical Q&A's from the DVC community.
+  This month: remote storage integration, hyperparameter tuning,
```

Comment:
can we revisit a bit the list of items? it doesn't look relevant

Reply:
Updated those!